### PR TITLE
Create umip-10

### PR DESCRIPTION
**Pull request by claytonroche**
draft for umip 10

### CODE REVIEW

**Comment by mrice32** [comment]
@claytonroche Same issue here as UMIPs 8 and 9.

File: UMIPs/umip-10.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+
+
+## Implementation
+
+This change has no implementation other than the two aforementioned governor transactions that will be proposed.
+
+## Security considerations
+Since WETH is a persistently valuable ERC20 token, including it as a collateral currency should impose no additional risk to the protocol.
```
Comment (nicholaspai) [inline UMIPs/umip-10.md:39]: Not neccessary to explicitly state this, but it's worth checking that the WETH contract is not this list of ERC20's that have some vulnerabilities, ranging from minor to major: https://github.com/sec-bit/awesome-buggy-erc20-tokens

File: UMIPs/umip-10.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Motivation
+ETH is the most liquid currency on Ethereum. Many users on Ethereum like to borrow against their ETH to get leverage on their ETH. At the time of writing, over 1.9mm ETH are locked in MakerDAO in this fashion. To allow synthetic tokens created with the EMP to take advantage of this liquidity and desire for leverage, WETH (ETH wrapped in an ERC20) seems like an important and safe addition to Dai as the second approved collateral currency.
+
+WETH as collateral is expected to have a variety of deployments.  The timing for adding it now, and the immediate application, is for use with USDETH, which will enable the creation of yUSD, a yielding dollar token.  This price identifier is described in UMIP 6.
+
+
+## Technical Specification
+To accomplish this upgrade, two changes need to be made:
+- The WETH address, [0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2](https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2), needs to be added to the collateral currency whitelist introduced in UMIP-8.
+- A final fee of 0.1 needs to be added for the WETH in the Store contract.
```
Comment (chrismaree) [inline UMIPs/umip-10.md:23]: Could we add a bit of a rational as to why 0.1 is chosen as the final fee?

File: UMIPs/umip-10.md (hunk 1 above)
Comment (mrice32) [reply UMIPs/umip-10.md:39]: Great point. We should add that link to the UMIP.

Comment (mrice32) [reply UMIPs/umip-10.md:31]: Done -- I checked, stated it, and linked it.

File: UMIPs/umip-10.md (hunk 2 above)
Comment (mrice32) [reply UMIPs/umip-10.md:20]: @chrismaree I added a blurb in the Rationale section. PTAL.

Comment (chrismaree) [reply UMIPs/umip-10.md:20]: Looks perfect, thanks!

**Pull request merged**